### DISCUSSION TITLE
State Diagram (in project main page)

### DISCUSSION THREAD

**Discussion by modem-man-gmx** [hoylabs/OpenDTU-OnBattery#160] [Q&A] [answered]
Hi Helge,
mit welchem Tool erzeugst Du das State-Diagramm? Finde ich cool und vor allem sehr intuitiv. Will ich auch benutzen, wenn das irgendwo verfügbar ist.

**Comment by helgeerbe** [comment, marked as answer]
@modem-man-gmx das ist [DrawIo](https://drawio-app.com). Entweder im Webbrowser Zeichnen, oder in visualStudio Code die Erweiterung installieren.
